src/parsers/power_system_table_data.jl

```diff
@@ -790,6 +814,40 @@ function calculate_variable_cost(data::PowerSystemTableData, gen, cost_colnames,
     return var_cost, fixed, fuel_cost
 end
 
+function calculate_variable_cost(
+    data::PowerSystemTableData,
+    gen,
+    cost_colnames::_CostPointColumns,
+    base_power,
+)
+    vom = isnothing(gen.variable_cost) ? 0.0 : gen.variable_cost
+
+    var_cost = [(getfield(gen, c), getfield(gen, mw)) for (c, mw) in cost_colnames.columns]
+    var_cost = unique([
+        (tryparse(Float64, string(c[1])), tryparse(Float64, string(c[2])))
```

Comment:
What will happen if `tryparse` returns `nothing`? Should this be `parse` instead? Might want to reconsider other uses of `tryparse` in this file.

Reply:
`tryparse` returns a `nothing` if it fails while `parse` returns an error. In this case, there are sometimes columns with `"NA"` or some other non numeric value that we want to just ignore. So, for this case, I think it achieves the desired outcome. But I'll double check the other usages.